test/api/pull/index.test.ts

```diff
@@ -17,8 +20,47 @@ const privateKey = fs.readFileSync(
   "utf-8"
 );
 
+const collaborator = function (
+  login: string,
+  permissions: {
+    pull: boolean;
+    push: boolean;
+    admin: boolean;
+  },
+  rest?: any
```

Comment:
Why we need this?

Reply:
For other test data which need to customize the other attributes.

Reply:
It seems you do not use it?

Reply:
Indeed.